drivers/input/keyboard/brain-kbd-i2c.c

```diff
@@ -24,6 +23,18 @@
 
 #define BK_IS_PRESSED(val) ((~val & 0x40) >> 6)
 
+#define BK_IS_SWITCH(val)   (((val) & 0x80)!=0)
+#define BK_SW_CODE(val)     (((val)>>1)&0x1F)
+#define BK_SWITCH_ON(val)   (((val)&1)==0)
+
+enum{
+	BK_SW_LCD_DIR1  = 3,
+	BK_SW_UNKNOWN4,
+	BK_SW_USB_VBUS,
+	BK_SW_LCD_CLOSE,
+	BK_SW_LCD_DIR2
+};
```

Comment:
DIR1、DIR2 といった番号だと意味が不明確なので、より説明的な名付けが望ましいです 👍 
わかりやすさ優先なので、長い名前で構いません。
以下は勘違いしてたら修正していただきたいのですが、例えばこんなのとか。

```suggestion
enum {
	BK_SW_LCD_TRANSFORMING_TO_TABLET = 3,
	BK_SW_UNKNOWN4,
	BK_SW_USB_VBUS,
	BK_SW_LCD_FULLY_TRANSFORMED,
	BK_SW_LCD_TRANSFORMING_TO_CLOSED
};
```

Reply:
DIR1とDIR2の意味がよくわからず保留にしておりました。
正しいかどうかは分かりませんが、ご提案いただいた内容に変更します。